### PR TITLE
chore: Enable field alignment go vet check

### PR DESCRIPTION
**Pull request by patriciareinoso**
Go vet checks were enabled in https://github.com/omec-project/gnbsim/pull/116
(but they were disable in following commits)
This PR re enables all go vet analyzers and fixes `fieldalignment` analyzer

### CODE REVIEW

**Comment by thakurajayL** [comment]
@gab-arrobo  can you please review this.
@patriciareinoso  could you pleae resolve conflict.

**Comment by gab-arrobo** [comment]
> @gab-arrobo can you please review this.

Will do

File: gnodeb/context/gnbcpue.go [hunk 1]
```diff
@@ -15,16 +15,9 @@ import (
...
-	Amf         *GnbAmf
-	Gnb         *GNodeB
-
-	// TODO: Sync map is not needed as it is handled single threaded
-	GnbUpUes sync.Map
-
-	WaitGrp sync.WaitGroup
+	Supi string
+	Amf  *GnbAmf
+	Gnb  *GNodeB
```
Comment (gab-arrobo) [inline gnodeb/context/gnbcpue.go:20]: Why not putting the pointers together (including `Log *logrus.Entry`)? Also, probably we can remove the text `// logger`, the variable itself seems self-explanatory, what do you think?

File: gnodeb/context/gnbupf.go [hunk 2]
```diff
@@ -18,8 +18,7 @@ const GTP_U_PORT int = 2152
 
 // GnbUpf holds the UPF context
 type GnbUpf struct {
-	UpfAddr     *net.UDPAddr
-	UpfIpString string
+	UpfAddr *net.UDPAddr
```
Comment (gab-arrobo) [inline gnodeb/context/gnbupf.go:21]: Something similar to my previous comment, why not putting variables of the same type together? For example:
```go
	UpfAddr  *net.UDPAddr
	GnbUpUes *GnbUeDao
	Log      *logrus.Entry
```

File: gnodeb/context/gnbupue.go [hunk 3]
```diff
@@ -14,15 +14,10 @@ import (
...
-	DlTeid           uint32
-	UlTeid           uint32
-	Snssai           models.Snssai
-	Upf              *GnbUpf
-	Gnb              *GNodeB
-	PduSessType      models.PduSessionType
-	QosFlows         map[int64]*ngapType.QosFlowSetupRequestItem
-	LastDataPktRecvd bool
+	Upf         *GnbUpf
+	Gnb         *GNodeB
```
Comment (gab-arrobo) [inline gnodeb/context/gnbupue.go:18]: Same comment as before
```go
	Upf         *GnbUpf
	Gnb         *GNodeB
	Log         *logrus.Entry
```

**Review by gab-arrobo — COMMENTED** [review]
Please taka look at the comments and let me know what you think... This is someting that could be done in several files because `/* logger */` can be removed and `Log *logrus.Entry` could be put next to other variables of the same type

**Comment by patriciareinoso** [comment]
> Please taka look at the comments and let me know what you think... This is someting that could be done in several files because `/* logger */` can be removed and `Log *logrus.Entry` could be put next to other variables of the same type

I am always eager to remove unnecessary comments. I moved all the log fields next to other pointers

**Comment by gab-arrobo** [comment]
@patriciareinoso, please resolve conflicts. Thanks!

**Review by gab-arrobo — APPROVED** [review]
+1 (tested changes with AiaB)

**Pull request merged**